cluster/saltbase/salt/kubelet/default

```diff
@@ -151,12 +151,11 @@
   {% endif -%}
 {% endif -%}
 
-{% set configure_hairpin_mode = "--configure-hairpin-mode=true" -%}
-{% if pillar.get('hairpin_mode', '').lower() == 'false' %}
-  {% set configure_hairpin_mode = "--configure-hairpin-mode=false" %}
+{% set hairpin_mode = "" -%}
```

Comment:
I think the default should depend on whether kubeproxy is in userspace mode(none) or iptables mode(promiscous)
If this is making the salting tougher, its ok to leave it the way you have it.